scylladb/sphinx-scylladb-theme
# Replace recommonmark for Myst parser

Related issue #168 

Replaces recommonmark for Myst parser for new projects.

## How to test

1. Clone this pull request

2. Build the docs.

3. The following page written in Markdown should render without errors: http://127.0.0.1:5500/configuration/markdown/

   ![image](https://github.com/scylladb/sphinx-scylladb-theme/assets/9107969/774400c7-ee94-4cde-a25f-b8c4ff993d5e)
